open-metadata-resources/open-metadata-deployment/compose/tutorials/.env

```diff
@@ -6,6 +6,10 @@ egeria_version=2.8-SNAPSHOT
 #
 # Version of egeria-react-ui to use
 egeria_react_ui_version=2.7.0-rc.0
+
+# Version of egeria-ui to use
```

Comment:
👍

Reply:
I've set to latest for now.. this needs some refinement moving forwards. for the next release we can consider setting it to a fixed value after the branch - open to whatever you are comfortable with.